src/pages/Pool/PositionPage.tsx

```diff
@@ -346,7 +346,32 @@ const useInverter = ({
   }
 }
 
-export function PositionPage() {
+const INVALID_STATE = (
```

Comment:
To be more idiomatic, could you render this as a function? I'm not sure what the perf implications are of fn vs const, but this seems non-standard.

```suggestion
function PositionNetworkMismatch() {
```